log.js

```diff
@@ -11,15 +11,19 @@ const logLevels = ['fatal', 'error', 'warn', 'info', 'debug'];
 const logStack = getenv.bool('LOG_STACK', true);
 function forward(target, prefix) {
   return (format, ...args_) => {
-    const placeholders =
-      isString(format) ? (format.match(/%[sdifj]/g) || []).length : 0;
+    const placeholders = isString(format) ?
+      (format.match(/%[sdifj%]/g) || [])
```

Comment:
node 8.4.0 added `%o` and `%O` specifiers, see https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V8.md#8.4.0

Reply:
Fixed